### PR TITLE
QOL changes for Nav

### PR DESCRIPTION
**Pull request by peopledrivemecrazy**
Nav component now has active link highlighting which page user is at.

Changed the `/cheat` route to `/cheatsheet`, this seems appropriate.

### CODE REVIEW

**Comment by peopledrivemecrazy** [comment]
image

* Highlights which route is active.
* `{#each}` takes care of the links.
* `.active` has the same colour has the hover.

File: src/components/layout/Nav.svelte [hunk 1]
```diff
@@ -1,5 +1,15 @@
 <script>
-  import { prefetch } from "@roxi/routify";
+  import { prefetch, isActive, url } from "@roxi/routify";
+  console.log($isActive("/cheat"));
```
Comment (geoffrich) [inline src/components/layout/Nav.svelte:3]:
```suggestion
```

**Review by diepoe — APPROVED** [review]
LGTM and thanks for the nice feature

File: src/components/layout/Nav.svelte (hunk 1 above)
Comment (diepoe) [reply src/components/layout/Nav.svelte:3]: oh sorry my fault

Comment (peopledrivemecrazy) [reply src/components/layout/Nav.svelte:3]: Oops, @geoffrich thanks, Fixing them.

**Pull request merged**